README.md

```diff
@@ -26,7 +26,7 @@ Start a development instance of Nomad:
 
 Run the required services in Nomad:
 
-    cue export -e jobs.dev | nomad job run -
+    cue export -e jobs.dev -t env=local | nomad run -
```

Comment:
`local` is the [default](https://github.com/input-output-hk/cicero/blob/9be2c651dc0b2948d51f876b8c91c269eb38f291/deploy.cue#L3).

Why was it necessary to add this standup today?

Reply:
I just copied the command from our meeting chat and didn't realize this was a default.